src/test/resources/test-application.properties

```diff
@@ -6,3 +6,5 @@ spring.datasource.platform=postgresql
 spring.jpa.properties.hibernate.jdbc.lob.non_contextual_creation=true
 spring.jpa.hibernate.ddl-auto=none
 spring.jpa.hibernate.ddl=false
+
+datastore.default.path=d:\\epam\\report-portal\\data\
```

Comment:
can we use some temp dir here?

Reply:
sure, this is test properties file, will update property to datastore.default.path=/data/storage as in main applicaiton.properties.